bodybank/character_generator/views.py

```diff
@@ -14,12 +19,38 @@ def get_ego(request):
             ego = form.save(commit=False)
             ego.user = request.user
             ego.save()
-            return redirect('thanks')
+            return redirect('manage_egos_list')
             
     else:
         form = EgoForm()
 
     return render(request, 'ego.html', {'form': form})
 
+@login_required
+def update_ego(request, pk):
+    ego = get_object_or_404(Ego, pk=pk)
+    form = EgoForm(request.POST or None, instance=ego)
+    if form.is_valid():
+        form.save()
+        messages.success(request, 'Updated Ego')
```

Comment:
I really like the messages framework (specially when rendered with django bootstrap)